.travis.yml

```diff
@@ -1,5 +1,6 @@
 language: node_js
 node_js:
-  - "6"
-  - "8"
-  - "10"
+  - 6
```

Comment:
Can you drop Node 6?

Reply:
We have already dropped nodejs 6 support in https://github.com/mcollina/aedes-packet/blob/master/.travis.yml :)